### PR TITLE
Add Region To PubkeyResource Model

### PR DESCRIPTION
**Pull request by avitova**
We need a region for a pubkey. I felt this should belong to PubkeyResource. Correct me if I am wrong. I was not sure if the region should be required, but I felt like that should not be a problem. The value is prefilled for now.

### CODE REVIEW

File: internal/db/migrations/004_pubkeys.sql [hunk 1]
```diff
@@ -17,7 +17,8 @@ CREATE TABLE pubkey_resources
   pubkey_id BIGINT NOT NULL REFERENCES pubkeys(id),
   provider INTEGER NOT NULL CHECK (valid_provider(provider)),
   source_id TEXT NOT NULL,
-  handle TEXT NOT NULL
+  handle TEXT NOT NULL,
+  region TEXT NOT NULL
```
Comment (ezr-ondrej) [inline internal/db/migrations/004_pubkeys.sql:21]: I'm so sorry you've had to be the first, but we'll need to do a proper migration now that we are in stage :(

Comment (lzap) [reply internal/db/migrations/004_pubkeys.sql:21]: To elaborate, create a new SQL file with a new sequence numer and then do ALTER TABLE command to add a new column.

File: internal/dao/sqlx/pubkey_resource_dao.go [hunk 2]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	createPubkeyResource            = `INSERT INTO pubkey_resources (pubkey_id, provider, source_id, handle, tag) VALUES ($1, $2, $3, $4, $5) RETURNING id, tag`
+	createPubkeyResource            = `INSERT INTO pubkey_resources (pubkey_id, provider, source_id, handle, tag, region) VALUES ($1, $2, $3, $4, $5, 'us-east-1') RETURNING id, tag`
 	deletePubkeyResourceById        = `DELETE FROM pubkey_resources WHERE id = $1`
```
Comment (lzap) [inline internal/dao/sqlx/pubkey_resource_dao.go:16]: You also need to modify the create function, it need to set it.

**Review by ezr-ondrej — APPROVED** [review]
LGTM, thanks @avitova :+1:

@lzap? :)

File: internal/db/migrations/007_add_region.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pubkey_resources
+ADD region text;
```
Comment (lzap) [inline internal/db/migrations/007_add_region.sql:2]: We are just about to merge a huge PR that adds 007 and 008 migration files, can you rebase and rename this to 009?

File: internal/models/pubkey_resource.go [hunk 4]
```diff
@@ -26,6 +26,9 @@ type PubkeyResource struct {
 
 	// Resource handle (id). Format is provider-dependant. Required.
 	Handle string `db:"handle" json:"handle"`
+
+	// Required.
```
Comment (lzap) [inline internal/models/pubkey_resource.go:30]:
```suggestion
	// Region name. This is provider-dependant. Required for providers which don't have global public keys.
```

**Review by lzap — CHANGES_REQUESTED** [review]
You need to rebase because of migration conflict.

It would be nice to create custom types for both `Handle` and `Region` that would have `Provider` and `Value` fields and would support custom serialization to DB as a concatenated string (e.g. `provider-xxx`). But it's just a suggestion for future improvements, it's fine for now.

**Comment by avitova** [comment]
I forgot to rebase, now this should be ready.

**Pull request merged**